.papr.yml

```diff
@@ -0,0 +1,15 @@
+branches:
+  - master
+  - auto
+  - try
```

Comment:
Hmm, not sure why you need anything other than 'master'.

Reply:
Me neither.  I will remove auto and try.

Reply:
Those are for homu.

Reply:
(Mostly we just use the `auto` branch - when doing a merge, it pushes there first and waits for PAPR to report)

Reply:
What's a "homu"? Can you eat it? 😀 

@rhatdan I checked the docs, this is just the branches to test, so just 'master' is appropriate here since none of the other branches exist.

/me still wonders what a homu is

Reply:
It's linked from https://github.com/projectatomic/papr#more-details-about-project-atomic-ci-services

Reply:
@cevich Homu will create those branches when doing a merge.

Reply:
Ooohhh, https://github.com/servo/homu/ it's a service.  gotcha, kk, I'll put those back and add a comment 😀